XML/TestCases/Other.xml

```diff
@@ -76,4 +76,22 @@
       <SetupType>long-term</SetupType>
     </SetupConfig>
   </test>
+  <test>
+    <testName>NTTTCP-MONITOR-THROUGHPUT</testName>
+    <testScript>NTTTCP-MONITOR-THROUGHPUT.ps1</testScript>
+    <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\ntttcp_monitor_throughput.sh</files>
+    <Platform>Azure</Platform>
+    <TestParameters>
+      <param>seconds_of_runtime=NTTTCP_MONITOR_SECS_RUNTIME</param>
+      <param>monitor_interval=NTTTCP_MONITOR_INTERVAL</param>
+      <param>tolerance=NTTTCP_MONITOR_TOLERANCE</param>
+      <param>ntttcp_version=master</param>
+    </TestParameters>
+    <Category></Category>
```

Comment:
Can you fill out Category and Area?

Reply:
we don't want this case to be selected by mistake as running with generic scope selection (by Category, or by Area only) Because this is a pretty long time running case, which need manually to be taken care of. So we suspect this should be selected only by testname.

Reply:
Understand. Can you add your comment inside of XML file? It will answer of future questions,